packages/paste-core/components/avatar/src/types.ts

```diff
@@ -6,15 +6,60 @@ import type { HTMLPasteProps } from "@twilio-paste/types";
 export type ColorVariants = "default" | "decorative10" | "decorative20" | "decorative30" | "decorative40";
 export type AvatarVariants = "user" | "entity";
 
-export type AvatarProps = HTMLPasteProps<"div"> &
-  Pick<BoxProps, "element"> & {
-    name: string;
-    size?: IconSize;
-    icon?: React.FC<React.PropsWithChildren<GenericIconProps>>;
-    src?: string;
-    color?: ColorVariants;
-    variant?: AvatarVariants;
-  };
+export type AvatarProps = HTMLPasteProps<"div"> & {
+  /**
+   * Used to determine the initials, alt text (if using an image), or title (if using an icon) of the Avatar.
+   *
+   * @type {string}
+   * @memberof AvatarProps
+   */
+  name: string;
+  /**
+   *
+   * @default sizeIcon70
+   * @type {IconSize}
+   * @memberof AvatarProps
+   */
+  size?: IconSize;
+  /**
+   * Pass a Paste Icon to display in the Avatar
+   *
+   * @type {React.FC<React.PropsWithChildren<GenericIconProps>>}
+   * @memberof AvatarProps
+   */
+  icon?: React.FC<React.PropsWithChildren<GenericIconProps>>;
+  /**
+   * Pass an image path to display in the Avatar
+   *
+   * @default null
+   * @type {string}
+   * @memberof AvatarProps
+   */
+  src?: string;
+  /**
+   * Override the default color for decorative purposes
+   *
+   * @default default
+   * @type {ColorVariants}
+   * @memberof AvatarProps
+   */
+  color?: ColorVariants;
+  /**
+   *
+   * @default user
+   * @type {AvatarVariants}
+   * @memberof AvatarProps
+   */
+  variant?: AvatarVariants;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default "AVATAR"
+   * @type {BoxProps["element"]}
+   * @memberof AvatarProps
+   */
+  element?: BoxProps["element"];
+};
 
 export type AvatarContentProps = {
```

Comment:
I don't think this one is an exported component, but it is appearing on the docsite

Reply:
I exported it because we use it in the user-dialog package but yeah it's not ideal on the docs site